redaxo/src/core/fragments/core/footer.php

```diff
@@ -1,3 +1,4 @@
+<?php if ('login' !== rex_be_controller::getCurrentPage()): ?>
```

Comment:
Bin mir nicht ganz sicher, aber tendenziell würde ich eher das Fragment gar nicht erst laden auf der Loginseite.
Oder gibt es dafür einen Use case, mit diesem Fragment (durch Überschreiben) etwas anderes auf der Loginseite zu machen?

(Wobei ich gerade die Stelle auch nicht vor Augen habe, wo das Fragment geladen wird.)

Reply:
Ich würde das geladene Fragment begrüßen. Damit konnte man dann einfach noch ein Kundenlogo oder sowas einschleusen (zB mit be_branding)

Reply:
@gharlan Das passiert hier, und ich bin mir nicht sicher, ob man dort wirklich die Login-Page ausklammern möchte oder nicht lieber im Fragment selbst, so wie es gerade im PR enthalten ist:

https://github.com/redaxo/redaxo/blob/2c35e551e98bccc2747f523930347b4a9dc3aa2b/redaxo/src/core/layout/bottom.php#L23-L26

(Mein Bauchgefühl war, im Fragment selbst. Aber du hast schon recht, konsequenter wäre vermutlich vorm include.)

Reply:
Steuerungslogik in der view finde ich unpassend.

Sollte lieber in den aufrufenden code

Reply:
@gharlan @staabm Thx, habe es angepasst.